### ISSUE TITLE
Write tutorial: Sample applications ePub

### ISSUE THREAD

**Issue by jnehring** [freme-project/Documentation#111]
Sample applications, e.g.: enrich ePub metadata with a pipeline: e-Translate, e-Entity, e-Link

See [How to write a tutorial](https://github.com/freme-project/Documentation/issues/104)

**Comment by jnehring** [comment]
@pheyvaer is this done? I think we forgot this task. I propose to include it to the documentation we write before the review.

**Comment by pheyvaer** [comment]
@jnehring I'll have a look at it next week.

**Comment by jnehring** [comment]
It is very likely that the bug https://github.com/freme-project/e-Internationalization/issues/35 will not be fixed until end of the week so you should not wait for it.

**Comment by pheyvaer** [comment]
Also we have an issue here freme-project/Broker/issues/169](https://github.com/freme-project/Broker/issues/169). Does it make sense to write a tutorial then? Because at the moment, none of the services support (without issues) HTML in and out. And if we skip those services than the tutorial will be the same as this [one](http://api.freme-project.eu/doc/0.5/tutorials/html-to-epub.html).

**Comment by jnehring** [comment]
Right. This functionality does not seem to be stable enough for a tutorial.
